firmware/arduino/chessboard/circleFunction.ino

```diff
@@ -85,3 +77,105 @@ void makeCircle(int circle, int firstQuarter, int lastQuarter)
         }
     }
 }
+
+void calculateStepsPerSlope(){
+
+    int outerRadius, deltaR;
+
+    // Calculate the radius of the largest circle in eighth steps
+    outerRadius = MILLIMETERS_PER_SQUARE * STEPS_PER_MILLIMETER * 2;
```

Comment:
Change MILLIMETERS_PER_SQUARE to MILLIMETERS_PER_UNITSPACE, since there are two unit spaces per one piece space. Seems like you accounted for that with the 2 * though.